quality/detekt.yml

```diff
@@ -175,35 +176,16 @@ complexity:
   LongMethod:
     active: true
     threshold: 20
-  NestedBlockDepth:
-    active: true
-    threshold: 3
```

Comment:
I see what you're saying about the thresholds now. Can we make this larger so it isn't hard to fix? Maybe 4? It'd be nice if it automatically failed things we'd nit in review.